From fea106f29d34f69bbc8d31d41702f1377ef08426 Mon Sep 17 00:00:00 2001
From: Thomas Haller <thaller@redhat.com>
Date: Sat, 2 May 2020 22:39:53 +0200
Subject: [PATCH 1/1] nm-online: fix build of nm-online for missing
 libcsiphash.la

Fixes: e468b48ab7b8 ('nm-online: allow configuring timeout via NM_ONLINE_TIMEOUT environment')
(cherry picked from commit 69798fa6cd841c54915fdbee048ccd5765313a9b)
(cherry picked from commit 53476d821d3c66896bcdc0e6f26a60c0a184f469)
---
 Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index 6cb709b8d545..35564aa5c578 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -4289,6 +4289,7 @@ clients_nm_online_LDADD = \
 	shared/nm-libnm-aux/libnm-libnm-aux.la \
 	shared/nm-glib-aux/libnm-glib-aux.la \
 	shared/nm-std-aux/libnm-std-aux.la \
+	shared/libcsiphash.la \
 	$(GLIB_LIBS) \
 	$(NULL)
 
-- 
2.25.4

